frontend/src/cljs/imas_seamap/core.cljs

```diff
@@ -60,6 +60,8 @@
     :re-boot                              events/re-boot
     :ajax/default-success-handler         (fn [db [_ arg]] (js/console.log arg) db)
     :ajax/default-err-handler             (fn [db [_ arg]] (js/console.error arg) db)
+    :get-serialized-hashstate             [events/get-serialized-hashstate]
+    :got-serialized-hashstate             events/got-serialized-hashstate
```

Comment:
Ditto here.  "Serialised" isn't used anywhere else (it's "put-hash", etc).  It's literally true, but it's long and probably doesn't convey much more.